### PR TITLE
feat - user profile page

### PR DESCRIPTION
**Pull request by prachi-sahani**
This PR includes the following changes:
- user profile page without edit profile feature
  - user details
  - questions posted by the user
- hiding passwords in local storage 

Credentials for testing -
1) username: prachisahani password:123456
2) username: itsHamhere password: 123456

image

### CODE REVIEW

File: src/pages/Profile.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        p: { md: 0, xs: 2 },
+        gap: 2,
+        pr: { md: 2 },
```
Comment (Hamdrive) [inline src/pages/Profile.tsx:20]: seems redundant to have both overall padding and padding right for medium sized screens

**Review by Hamdrive — COMMENTED** [review]
- Really liked the implementation to add new posts, and also checkout profile page
- Overall work with TS and MUI looks great

**Review by junaidshaikh-js — APPROVED** [review]
Good work. Everything seems fine. 

Can be merged.

File: src/pages/Profile.tsx (hunk 1 above)
Comment (prachi-sahani) [reply src/pages/Profile.tsx:20]: I'll fix it in the next PR.

**Pull request merged**